README.md

```diff
@@ -291,6 +292,18 @@ Metrics are used to monitor the health of your fleet across components. Continue
 | node:node_net_utilisation:sum_irate                                      |         :x:        |         :x:        |         :x:        |         :x:        |
 | 'node_namespace_pod:kube_pod_info:'                                      |         :x:        |         :x:        |         :x:        |         :x:        |
 
+### Server foundation metrics
+
+| Metric Name                                                                  | ACM 2.5            | ACM 2.6            | ACM 2.7            | ACM 2.8            |
+|------------------------------------------------------------------------------|--------------------|--------------------|--------------------|--------------------|
+| acm_managed_cluster_labels                                                   |         :x:        |         :x:        | :white_check_mark: | :white_check_mark: |
+| acm_managed_cluster_info                                                     |         :x:        |         :x:        |         :x:        | :white_check_mark: |
```

Comment:
acm_managed_cluster_info was most definitely exposed in ACM 2.6 if not earlier. It was being sent to telemeter.

Reply:
Should we include 2.6, and maybe even 2.5, @zhiweiyin318 and @elgnay ?

Reply:
great!

Reply:
so @cadawson lets update `acm_managed_cluster_info` to have white_check_mark across all columns of the table then

Reply:
Made that change. Thanks!